ceno_zkvm/src/bin/e2e.rs

```diff
@@ -41,6 +43,11 @@ struct Args {
     /// The preset configuration to use.
     #[arg(short, long, value_enum, default_value_t = Preset::Ceno)]
     platform: Preset,
+
+    /// The private input or hints. This is a raw file mounted as a memory segment.
+    /// Zero-padded to the next power-of-two size.
```

Comment:
Please be explicitly in which direction we are padding.

As written elsewhere, left padding would be most useful to use.

Also as far as I can tell, we are zero-padding to the next power of two, end then 'inaccessible'-padding to fill up the total range?  We should probably mention the two different kinds of padding in the doc comment.

Reply:
Interesting idea of dynamic memory growing leftwards. Can you write an issue for it?

Reply:
See https://github.com/scroll-tech/ceno/issues/628

Reply:
See also https://github.com/scroll-tech/ceno/issues/634